src/cli/sql.rs

```diff
@@ -110,6 +115,25 @@ pub async fn init(
 		}
 	};
 
+	if !hide_welcome {
+		eprintln!(
+			"
+#
+#  Welcome to the SurrealDB SQL Shell
+#
+#  How to use this shell:
+#    - Different statements within a query should be separated by a (;) semicolon.
+#    - To create a multi-line query, end your lines with a (\\) backslash, and press enter.
```

Comment:
I think it's important to let users know that this is only possible when running with `--multi`.

Reply:
It would be even better to only add that line when running in `--multi` mode.